app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
```

Comment:
tìm hiểu dùng find_by thay vì find nhé, và cho anh biết 2 cái này khác nhau như thế nào
bonus, thêm xử lý nếu không tìm thấy user

Reply:
rep:
 + find: chi tim kiem theo ID. Neu ko tim thay se tra ve Exception va co the gay ra hien tuong crash sever.
 + find_by: cho phep tim kiem theo ID, gia tri trong bang. Neu ko tim thay se tra ve nil, ko xay ra hien tuong crash.